hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/WriteStatus.java

```diff
@@ -61,6 +61,8 @@ public class WriteStatus implements Serializable {
 
   private final List<Pair<HoodieRecordDelegate, Throwable>> failedRecords = new ArrayList<>();
 
+  // true if this WriteStatus refers to a write happening in metadata table.
+  private final boolean updatesMetadataTable;
```

Comment:
How is this used?  Is there any other way instead of storing the info here?

Reply:
we went over this when we synced up f2f. within SparkRDDWriteClient.commit() method, 
where we would dereference the dag and collect the write status to driver. 
and bucketize as data table write status and metadata table write status.

Reply:
I prefer the original name: isMetadataTable

Reply:
IMO, I don't find any major upside to either of the names. lets just move on. Just don't wanna keep dragging the PR on this.